src/Microsoft.Azure.WebJobs/QueueAttribute.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Azure.WebJobs
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes")]
     [AttributeUsage(AttributeTargets.Parameter)]
     [DebuggerDisplay("{QueueName,nq}")]
-    public class QueueAttribute : Attribute
+    public class QueueAttribute : StorageAccountAttribute
```

Comment:
Adding the new StorageAttribute base class would be non breaking and would give us all 3. All the types make sense together, and in the next major version, we can simply remove the Parameter level usage from StorageAccountAttribute - it then stays around for class/method level overrides.